dataduct/config/config_actions.py

```diff
@@ -32,7 +32,7 @@ def sync_from_s3(filename):
     text = s3_file.text
 
     if filename is None:
-        print text
+        raise ValueError('Filename for config sync must be provided')
```

Comment:
The thing is that pygraphviz doesn't support a show command so it creates an inconsistency between the different file arguments. Let me just try to patch pygraphviz if I can in some other diff.